support/Procfile

```diff
@@ -1,4 +1,3 @@
-web: support/builder_web.sh
```

Comment:
My thought removing this was that there should be one preferred way of running the UI.  Having it run in the VM feels like us telling developers "this is the way to do it" and that feels wrong to me.  I know you and I both ignored this running in the VM and ran the web stack on our machines while working on these features, which tells me that's how it should be done.  I frequently want to restart the web stack when debugging frontend issues but I don't need my backend stack to get restarted too.

All that said, this is just my opinion, and I'm happy to hear feedback from everyone on what their preference for this is.